vaadin-notification-flow-parent/vaadin-notification-flow-integration-tests/src/main/java/com/vaadin/flow/component/notification/tests/NotificationWithClassNamesPage.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2000-2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.notification.tests;
+
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.notification.Notification;
+import com.vaadin.flow.router.Route;
+
+@Route("vaadin-notification/notification-class-names-test")
+public class NotificationWithClassNamesPage extends Div {
+
+    public NotificationWithClassNamesPage() {
+        final Notification notification = new Notification();
+        notification.addClassName("custom");
+
+        Button addClass = new Button("Add a class",
+                event -> notification.addClassName("added"));
+        addClass.setId("add-notification-btn");
```

Comment:
When reading this without enough context, I would assume the button calls `add(notification)`.
Let's make it more explicit by using something like `add-notification-class`.

I'd suggest to also update `clear-notification-btn` below accordingly.